lib/lambda_ethereum_consensus/fork_choice/store.ex

```diff
@@ -89,6 +94,11 @@ defmodule LambdaEthereumConsensus.ForkChoice.Store do
     {:reply, values, state}
   end
 
+  @impl GenServer
+  def handle_call({:get_current_status_message}, _from, state) do
```

Comment:
nit. We can have just an atom here
```suggestion
  def handle_call(:get_current_status_message, _from, state) do
```